ApiReview/API.Polly.Extensions/NoDocs/Microsoft.Extensions.DependencyInjection/PollyServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Assembly 'Polly.Extensions'
+
+using System;
+using Polly;
+using Polly.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
This should not be in a Microsoft.* namespace.

Reply:
It's now Polly.Extensions.DependencyInjection